kura/examples/org.eclipse.kura.example.gpio/OSGI-INF/gpio.xml

```diff
@@ -26,6 +26,7 @@
    <property name="service.pid" type="String" value="org.eclipse.kura.example.gpio.GpioComponent"/>
    <service>
        <provide interface="org.eclipse.kura.example.gpio.GpioComponent"/>
```

Comment:
I am happy to remove this. Now it doesn't make sense to me either. But I didn't want to merge any change which could be seen as "breaking things".

Just merge this one and I will follow up with a new PR, removing unnecessary exports.